numpy/ma/core.py

```diff
@@ -4157,7 +4157,7 @@ def __repr__(self):
             suffix=',')
 
         if self._fill_value is None:
-            self.fill_value  # initialize fill_value
+            self.fill_value  # initialize fill_value  # noqa: B018
```

Comment:
it's probably out of scope, but turning `_fill_value` into a `@cached_property` would simplify the logic in the `fill_value` getter and setter, and would avoid this check.

Reply:
Perhaps leave that for a later PR. I haven't ever used `@cached_property`.